test/views/authenticated/AlreadyClaimingFREDifferentAmountsViewSpec.scala

```diff
@@ -30,32 +30,32 @@ class AlreadyClaimingFREDifferentAmountsViewSpec extends OptionsViewBehaviours[A
 
   val form = new AlreadyClaimingFREDifferentAmountsFormProvider()()
 
-  val application = applicationBuilder(userAnswers = Some(fullUserAnswers)).build()
+  val application = applicationBuilder(userAnswers = Some(currentYearFullUserAnswers)).build()
 
   val view = application.injector.instanceOf[AlreadyClaimingFREDifferentAmountsView]
 
   def applyView(form: Form[_]): HtmlFormat.Appendable =
     view.apply(
       form,
       NormalMode,
-      fullUserAnswers.get(ClaimAmountAndAnyDeductions).get,
-      fullUserAnswers.get(FREAmounts).get
+      currentYearFullUserAnswers.get(ClaimAmountAndAnyDeductions).get,
+      currentYearFullUserAnswers.get(FREAmounts).get
     )(fakeRequest, messages)
 
   def applyViewMultipleYears(form: Form[_]): HtmlFormat.Appendable =
     view.apply(
       form,
       NormalMode,
-      fullUserAnswers.get(ClaimAmountAndAnyDeductions).get,
+      currentYearFullUserAnswers.get(ClaimAmountAndAnyDeductions).get,
       Seq(FlatRateExpenseAmounts(Some(FlatRateExpense(100)), TaiTaxYear()), FlatRateExpenseAmounts(Some(FlatRateExpense(100)), TaiTaxYear().prev))
     )(fakeRequest, messages)
 
   def applyViewWithAuth(form: Form[_]): HtmlFormat.Appendable =
     view.apply(
       form,
       NormalMode,
-      fullUserAnswers.get(ClaimAmountAndAnyDeductions).get,
-      fullUserAnswers.get(FREAmounts).get
+      currentYearFullUserAnswers.get(ClaimAmountAndAnyDeductions).get,
+      currentYearFullUserAnswers.get(FREAmounts).get
     )(fakeRequest.withSession(("authToken", "SomeAuthToken")), messages)
 
```

Comment:
Could we possible create one apply view for these three? e.g. 

  def applyViewTest(
                     form: Form[_],
                     freAmounts: Seq[FlatRateExpenseAmounts] = currentYearFullUserAnswers.get(FREAmounts).get,
                     authType: Boolean = false): HtmlFormat.Appendable =
    view.apply(
      form,
      NormalMode,
      currentYearFullUserAnswers.get(ClaimAmountAndAnyDeductions).get,
      freAmounts
    )(if (withAuth) fakeRequest.withSession("authToken", "SomeAuthToken") else fakeRequest, messages)